setup.py

```diff
@@ -13,7 +13,7 @@
         long_description="""Common loaders for MIR datasets.""",
         keywords='mir dataset loader audio',
         license='BSD-3-Clause',
-        install_requires=['tqdm', 'librosa >= 0.7.0', 'numpy>=1.16', 'six', 'jams', 'requests'],
+        install_requires=['tqdm', 'librosa >= 0.7.0', 'numpy>=1.16', 'six', 'jams', 'requests', 'dali-dataset'],
```

Comment:
move this out of install_requires and just keep it in extras_require, but as a separate key, e.g.
```
extras_require = {
   'tests': ...
   'docs': ...
   'dali':  ['dali-dataset']
}
```

And we should add instructions to the docs/readme explaining how to install it - `pip install mirdata[dali]`. We'll have to update the tests to install all the requirements... @drubinstein any thoughts?